MvvmCross.Android.Support/V7.RecyclerView/MvxRecyclerAdapter.cs

```diff
@@ -27,70 +26,52 @@ namespace MvvmCross.Droid.Support.V7.RecyclerView
     public class MvxRecyclerAdapter
         : Android.Support.V7.Widget.RecyclerView.Adapter, IMvxRecyclerAdapter, IMvxRecyclerAdapterBindableHolder
     {
-        private readonly IMvxAndroidBindingContext _bindingContext;
-
         private ICommand _itemClick, _itemLongClick;
         private IEnumerable _itemsSource;
         private IDisposable _subscription;
         private IMvxTemplateSelector _itemTemplateSelector;
 
-        protected IMvxAndroidBindingContext BindingContext => _bindingContext;
-
-        public MvxRecyclerAdapter() : this(MvxAndroidBindingContextHelpers.Current())
-        {
-        }
+        protected IMvxAndroidBindingContext BindingContext { get; }
 
-        public MvxRecyclerAdapter(IMvxAndroidBindingContext bindingContext)
+        public MvxRecyclerAdapter(IMvxAndroidBindingContext bindingContext = null)
```

Comment:
Doesn't Android need an empty ctor? Maybe just leave it as it was?

Reply:
ok